lua/gruvbox_crisp/groups.lua

```
@@ -0,0 +1,271 @@
+    ["@keyword"]        = { link = "Keyword" },
+    ["@keyword.conditional"] = { fg = p.kw_ctrl },
+    ["@keyword.repeat"]      = { fg = p.kw_ctrl },
+    ["@keyword.return"]      = { fg = p.kw_ctrl },
+    ["@keyword.exception"]   = { fg = p.kw_ctrl },
+    ["@keyword.import"]      = { fg = p.kw_ctrl },  -- Python import/from keywords
+    ["@keyword.directive"]   = { fg = p.preproc },
+    ["@operator"]       = { link = "Operator" },
+    ["@punctuation"]    = { fg = p.punct },
+    ["@punctuation.bracket"] = { fg = p.punct },
+    ["@punctuation.delimiter"] = { fg = p.punct },
+    ["@punctuation.special"]   = { fg = p.operator },
```

Luacheck (1.2.0)

[warning] value assigned to field '@punctuation.special' is overwritten on line 208 before use

(W314)